### PR TITLE
Update docs for release-0.15

### PR DESCRIPTION
**Pull request by cxbrowne1207**
*Issue #, if available:*

*Description of changes:*
This PR updates docs necessary for release. It also makes changes to indicate how to grab artifacts for a specific release, defaulting to v0.15.0

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cxbrowne1207** [comment]
/hold

wait for new amplify site to deploy via CFN and to build in Amplify

File: docs/content/en/docs/reference/artifacts.md [hunk 1]
```diff
@@ -28,37 +28,64 @@ However, see [Building node images]({{< relref "#building-node-images">}}) for i
...
 ### Bottlerocket OS images for Bare Metal
 
-Bottlerocket vends its Baremetal variant Images using a secure distribution tool called `tuftool`. Please refer to [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket image. You can also get the download URIs for Bottlerocket Baremetal images from the bundle release by running the following command:
+Bottlerocket vends its Baremetal variant Images using a secure distribution tool called `tuftool`. Please refer to [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket image. You can also get the download URIs for Bottlerocket Baremetal images from the bundle release by running the following commands:
+
+Using the latest EKS Anywhere version
+```bash
+EKSA_RELEASE_VERSION=$(curl -sL https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml | yq ".spec.latestVersion")
+```
+
```
Comment (jiayiwang7) [inline docs/content/en/docs/reference/artifacts.md:37]: Maybe add an "OR" ? same for all

**Review by jiayiwang7 — APPROVED** [review]

**Pull request merged**